aws/table_aws_amplify_app.go

```diff
@@ -0,0 +1,286 @@
+package aws
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/amplify"
+	"github.com/turbot/go-kit/helpers"
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsAmplifyApp(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_amplify_app",
+		Description: "AWS Amplify App",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("app_id"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"ValidationException"}),
+			},
+			Hydrate: getApp,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listApps,
+		},
+		GetMatrixItem: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "app_id",
```

Comment:
Please rearrange the columns in the below order -
The top 5 columns(Non-JSON) should be the primary/important ones like name, app_id, arn, create_time, and platform.
Next, all the columns, excluding the JSON ones, should be sorted.
All the JSON columns should be at last but before the Steampipe standard columns and in sorted order.

Reply:
I've rearranged. 
I placed my 6 most import columns at the top in level of importance. 
Following I had all remain fields that are not JSON listed in alphabetical order, followed by JSON fields. 
Finally the last fields are the `common` Turbot ones